infra/create-cluster/action.yml

```diff
@@ -22,6 +22,10 @@ inputs:
     description: Whether to wait for the cluster readiness
     required: false
     default: "false"
+  no-slack:
+    description: Whether to skip sending Slack messages for lifecycle events
+    required: false
+    default: "false"
```

Comment:
We should try to stick to the positive statements in boolean flags.

I recall discussion with Fred about naming `ROX_DONT_DISABLE_STH=FALSE` 😂

Reply:
In this case it's ok like it is, because there is a `--no-slack` flag in `infractl`...